package.json

```diff
@@ -11,7 +11,9 @@
     "prelint": "npm run format",
     "lint": "eslint --fix \"./src/**/*.{ts,js,vue}\"",
     "format": "prettier --write .",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
```

Comment:
We use bun now? Yarn.lock or bunlock must be removed

Reply:
It works fine, but no, we stick to npm.